.github/actions/update-project/update-project.sh

```diff
@@ -83,3 +96,61 @@ for package in projects/packages/*; do
 
 	cd $BASE
 done
+
+echo "Cloning folders in projects/plugins and pushing to Automattic package repos"
+
+# sync to read-only clones
+for plugin in projects/plugins/*; do
+	[ -d "$plugin" ] || continue # We are only interested in directories (i.e. plugins)
+
+	cd $BASE
+
+	# Only keep the plugin's name
+	NAME=${plugin##*/}
+
+	echo " Name: $NAME"
+
+	CLONE_DIR="__${NAME}__clone__"
+	echo "  Clone dir: $CLONE_DIR"
+
+	if [ "$NAME" == 'jetpack' ]; then
+		GIT_SLUG='jetpack-production'
+	else
+		GIT_SLUG="jetpack-${NAME}";
+	fi
+	# Check if a remote exists for that package.
+	$( git ls-remote --exit-code -h "https://github.com/automattic/${GIT_SLUG}.git" >/dev/null 2>&1 ) || continue
+	echo "  ${NAME} exists. Let's clone it."
+
+	# clone, delete files in the clone, and copy (new) files over
+	# this handles file deletions, additions, and changes seamlessly
+	git clone --depth 1 https://$API_TOKEN_GITHUB@github.com/automattic/$GIT_SLUG.git $CLONE_DIR
+
+	echo "  Cloning of ${NAME} completed"
+
+	cd $CLONE_DIR
+
+	find . | grep -v ".git" | grep -v "^\.*$" | xargs rm -rf # delete all files (to handle deletions in monorepo)
```

Comment:
Additionally, if a file is added to the special `.gitignore` after the initial commit, it won't ever be deleted here since `.gitignore` doesn't impact already tracked files. I feel there's probably a weakness there we should tighten.

As for the actual deletion, you're right. We're saved on this so far since it also `rm -rf` all folders.

Reply:
Tracking this via 1199714989172614-as-1199528688330363 during the reorg project.